images/dcos/Makefile

```diff
@@ -18,10 +18,9 @@ GIT_REF_MINOR = $(lastword $(subst -, ,$(GIT_REF)))
 GIT_REF_MAJOR = $(firstword $(subst -, ,$(GIT_REF)))
 ifneq ($(wildcard $(BUILD_DIR)),)
 	UNCLEAN_INFIX := -unclean
-	UNCLEAN_PREFIX := $(USER)-
 endif
 ifeq ($(KUBE_ROOT),)
-	VERSION := $(UNCLEAN_PREFIX)$(shell git describe --match $(GIT_REF_MAJOR) --always --tags --dirty)$(UNCLEAN_INFIX)-$(GIT_REF_MINOR)
+	VERSION := $(USER)$(shell git describe --match $(GIT_REF_MAJOR) --always --tags --dirty)$(UNCLEAN_INFIX)-$(GIT_REF_MINOR)
 else
 	VERSION := $(USER)-$(GIT_REF)-dev
```

Comment:
seems like the `$(USER)` line here should also be `$(UNCLEAN_PREFIX)` to avoid problems with an empty USER?